std/socket.d

```diff
@@ -2928,28 +2928,37 @@ public:
      */
     enum int ERROR = _SOCKET_ERROR;
 
+    private static int capToInt(size_t size) nothrow @nogc
```

Comment:
`@safe` is implicit (declared at the top of the module). `pure` makes no difference here because it is a private method called only by non-pure methods.